test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs

```diff
@@ -19,7 +20,12 @@ public class BuildIntegrationTest : MSBuildIntegrationTestBase
             Assert.FileExists(result, OutputPath, "SimpleMvc.dll");
             Assert.FileExists(result, OutputPath, "SimpleMvc.PrecompiledViews.dll");
 
-            Assert.BuildOutputContainsLine(result, $"SimpleMvc -> {Path.Combine(Path.GetFullPath(Project.DirectoryPath), OutputPath, "SimpleMvc.PrecompiledViews.dll")}");
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Darwin)
```

Comment:
Did you mean `!= Darwin`? Also we have `ConditionalFacts` you could use to skip this test on OSX

Reply:
I don't want to skip the whole test, just this one verification.

Reply:
Yeah turning it into a `!=` oops 😆